### PR TITLE
Migrate `removeBuilds.php` backend from XSL to Blade

### PR DESCRIPTION
**Pull request by williamjallen**
This PR is part of our ongoing effort to migrate away from XSL completely.

### CODE REVIEW

File: resources/views/admin/remove-builds.blade.php [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+            type="text"
+            size="2"
+            value="{{ $monthFrom }}"
+        >
+        <input
+            name="dayFrom"
+            type="text"
+            size="2"
+            value="{{ $dayFrom }}"
+        >>
```
Comment (josephsnyder) [inline resources/views/admin/remove-builds.blade.php:47]: The duplicate `>` here makes it to the window:
image

File: app/Http/Controllers/AdminController.php [hunk 2]
```diff
@@ -248,47 +248,29 @@ public function removeBuilds(): View|RedirectResponse
...
+            $build = DB::select("
                          SELECT id
                          FROM build
                          WHERE
-                             projectid=?
+                             projectid = ?
                              AND parentid IN (0, -1)
-                             AND starttime<=?||'-'||?||'-'||?||' 00:00:00'
-                             AND starttime>=?||'-'||?||'-'||?||' 00:00:00'
+                             AND starttime <= CAST(CONCAT(?, '-', ?, '-', ?, ' 00:00:00') AS timestamp)
```
Comment (josephsnyder) [inline app/Http/Controllers/AdminController.php:269]: I get a 500 error when trying to run the remove builds for a couple of days time:

image


Is this supposed to be `DATETIME`?

**Review by josephsnyder — CHANGES_REQUESTED** [review]
Most things look good but some fixes needed.  A small display issue which will be easy to fix, but also a 500 when attempting to remove the builds from 3 days of history

File: app/Http/Controllers/AdminController.php (hunk 2 above)
Comment (williamjallen) [reply app/Http/Controllers/AdminController.php:269]: This works in Postgres, but not in MySQL.  I'll push a fix.

**Review by josephsnyder — APPROVED** [review]
Great!  Works on the MySQL environment that I have now

**Pull request merged**